addons/base_vat/views/res_partner_views.xml

```diff
@@ -13,7 +13,7 @@
                 </xpath>
                 <xpath expr="//div[@name='vat_vies_container']" position="inside">
                     <xpath expr="//field[@name='vat']" position="move"/>
-                    <label for="vies_valid" invisible="not perform_vies_validation"/>
+                    <br /><label for="vies_valid" invisible="not perform_vies_validation"/>
```

Comment:
I do think there should be a way of doing this without using a hardcoded `<br/>` ; you could have a look at `o_address_format`, where this field replaces `vat`. The fields then seem to suffice and don't mention their respective labels. Alternatively, creating a group might give the structure you're looking for :)